cadc-util-fs/build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id 'java'
+    id 'maven'
+    id 'maven-publish'
+    id 'checkstyle'
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    maven {
+        url "https://maven.restlet.talend.com"
+    }
+}
+
+apply from: '../opencadc.gradle'
+
+sourceCompatibility = 1.8
+
+group = 'org.opencadc'
+
+version = '1.0.0'
+
+description = 'OpenCADC VOSpace utility file system library'
+def git_url = 'https://github.com/opencadc/vos'
+
+dependencies {
+    implementation 'org.opencadc:cadc-util:[1.9.0,)'
+    implementation 'org.opencadc:cadc-log:[1.0,)'
```

Comment:
cadc-log not needed by this lib - when making a new lib, always strip the dependencies down to the minimum

Reply:
Done